run.sh

```diff
@@ -3,12 +3,16 @@
 set -x
 set -e
 
-ARGS=(--name vertx-fib-demo -p 8080:8080 -p 9093:9093)
+if [ -z "$HTTP_PORT" ]; then
+    HTTP_PORT=8080
+fi
+
+ARGS=(--name vertx-fib-demo -p $HTTP_PORT -p 9093:9093)
```

Comment:
Last critical piece, I think: `-p 9999` should really be `-p 9999:9999`, for example.